lib/d3locales.js

```diff
@@ -0,0 +1,181 @@
+'use strict';
+
+// https://github.com/mbostock/d3/wiki/Localization
+// https://github.com/mbostock/d3/wiki/Time-Formatting#format
+// some are in node_modules/d3/src/locale
+
+
+var d3locales = { };
+
+d3locales.cs_CZ = {
+  decimal: ',',
+  thousands: '.',
+  grouping: [3],
+  currency: ['', 'CZK'],
+  dateTime: '%A %e %b %Y %X',
+  date: '%d.%m.%Y',
+  time: '%H:%M:%S',
+  periods: ['AM', 'PM'],
+  days: ['Neděle', 'Pondělí', 'Úterý', 'Středa', 'Čtvrtek', 'Pátek', 'Sobota'],
+  shortDays: ['Ne', 'Po', 'Út', 'St', 'Čt', 'Pá', 'So'],
+  months: ['Leden', 'Únor', 'Březen', 'Duben', 'Květen', 'Červen', 'Červenec', 'Srpen', 'Září', 'Říjen', 'Listopad', 'Prosinec'],
+  shortMonths: ['Led', 'Úno', 'Bře', 'Dub', 'Kvě', 'Čvn', 'Čvc', 'Srp', 'Zář', 'Říj', 'Lis', 'Pro']
+};
+
+d3locales.de_DE = {
+  decimal: ",",
+  thousands: ".",
+  grouping: [3],
+  currency: ["", " €"],
+  dateTime: "%A, der %e. %B %Y, %X",
+  date: "%d.%m.%Y",
+  time: "%H:%M:%S",
+  periods: ["AM", "PM"], // unused
+  days: ["Sonntag", "Montag", "Dienstag", "Mittwoch", "Donnerstag", "Freitag", "Samstag"],
+  shortDays: ["So", "Mo", "Di", "Mi", "Do", "Fr", "Sa"],
+  months: ["Januar", "Februar", "März", "April", "Mai", "Juni", "Juli", "August", "September", "Oktober", "November", "Dezember"],
+  shortMonths: ["Jan", "Feb", "Mrz", "Apr", "Mai", "Jun", "Jul", "Aug", "Sep", "Okt", "Nov", "Dez"]
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Mixed double and single quotes.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/1013683211/issues/source?bid=2452379&fileBranchId=2875367#l37)